content/en/docs/reference/tools/_index.md

```diff
@@ -29,6 +29,9 @@ to a Kubernetes cluster, troubleshoot them, and manage the cluster and its resou
 [Helm](https://helm.sh/) is a tool for managing packages of pre-configured
 Kubernetes resources. These packages are known as _Helm charts_.
 
+Beware Helm is a third party managed tool for managing packages of pre-configured
+Kubernetes resources, aka Kubernetes charts.
```

Comment:
“Beware”? Why the strong wording?

Some readers might take this as a cue to avoid Helm; is that what we want?

Reply:
@sftim would you like to suggest some substitute word?

Reply:
removed Beware word.